consensus/src/errors.rs

```diff
@@ -69,10 +69,10 @@ impl From<BlsSigError> for ConsensusError {
 
 #[derive(Debug, Error)]
 pub enum OperationError {
-    #[error("VST failed: {0}")]
-    FailedVST(StateTransitionError),
-    #[error("EST failed: {0}")]
-    FailedEST(StateTransitionError),
+    #[error("State transition creation failed: {0}")]
```

Comment:
I have a problem with "transition creation", is it transition object creation? Transition process creation? Somehow transition creation does not convey for me what it really is.

Reply:
thank you for your explanation, I think "building of transition" or maybe "transition attempt" would be clearer, but I am ok also with "creation"